### ISSUE TITLE
Error while indexing blueprints

### ISSUE THREAD

**Issue by Raik89** [hallgeirl/gd-item-search#16]
I am receiving the following error once the initial setup completes, then again on every startup or refresh when the "Indexing Blueprints" step is reached:

> An error occurred while loading settings. Details: Object reference not set to an instance of an object

This error also occurs on every version I've tried after v0.2.3 (that version still works for me, but the recent changes to character save file formats causes issues with it).

I have included my formula files here: [formulas.zip](https://github.com/hallgeirl/gd-item-search/files/2683590/formulas.zip)

**Comment by hallgeirl** [comment]
Thanks for providing the formulas file. I found the issue, a bit unsure of the reason WHY, but for some reason the target item for the blueprint for Aether clusters didn't exist in the item cache. I've made a workaround until I can get to the bottom of this. A new version, 0.2.10, should be ready in a few minutes where this at least won't crash the indexing.

**Comment by hallgeirl** [comment]
Ok, I think I figured out the root cause here...
http://www.grimdawn.com/forums/showthread.php?t=27732
Apparently, Aether clusters used to be a blueprint, but has been removed a while ago. The database entry for the aether cluster item has probably later been renamed, but since the blueprint doesn't actually exist anymore, the reference to said database entry has likely not been updated.

Anyhow, the fix has been released as version 0.2.10 now.
